pkg/ui/key.go

```diff
@@ -182,33 +207,29 @@ func ParseKey(s string) (Key, error) {
 			if 'a' <= k.Rune && k.Rune <= 'z' {
 				k.Rune += 'A' - 'a'
 			}
-			// Tab is equivalent to Ctrl-I and Ctrl-J is equivalent to Enter.
-			// Normalize Ctrl-I to Tab and Ctrl-J to Enter.
+			// Normalize Ctrl-I to Tab, Ctrl-J to Enter, and Ctrl-? to Backspace.
 			if k.Rune == 'I' {
 				k.Mod &= ^Ctrl
 				k.Rune = Tab
 			} else if k.Rune == 'J' {
 				k.Mod &= ^Ctrl
 				k.Rune = Enter
+			} else if k.Rune == '?' {
+				k.Mod &= ^Ctrl
+				k.Rune = Backspace
```

Comment:
I don't think typing `Ctrl-?` actually produces backspace in any modern terminal, so I'd rather leave this out.